.travis.yml

```diff
@@ -27,6 +28,10 @@ addons:
 before_install:
 - source ./ci/before_install.sh
 
+matrix:
+  fast_finish: true
+  allow_failures:
+  - rust: nightly
```

Comment:
Why was this necessary? I'd prefer not to allow failures unless there is a reason it is flaky right now.